dev_guide/builds.adoc

```diff
@@ -1465,18 +1464,20 @@ git:
     name: "<committer name>"
     email: "<committer e-mail>"
   message: "<commit message>"
-env:
+env: <1>
    - name: "<variable name>"
      value: "<variable value>"
 
 ----
-====
 
-Similar to the link:#buildconfig-environment[*BuildConfig* environment] variables, the environment variables
-defined in the above payload will be made available to your build.  When variable names in the above payload
-collide with the names of the link:#buildconfig-environment[*BuildConfig* environment] variables, the variables
-in the payload will take precedence.  By default, environment variables passed via webhook are ignored.
-You must set `*allowEnv*` to true on the webhook definition to enable this behavior.
+<1> Similar to the link:#buildconfig-environment[*BuildConfig* environment]
+variables, the environment variables defined here will be made available to your
+build. When these variables collide with the
+link:#buildconfig-environment[*BuildConfig* environment] variables, these
+variables take precedence. By default, environment variables passed via webhook
+are ignored. Set `*allowEnv*` to `*true*` on the webhook definition to enable
+this behavior.
```

Comment:
It took me a bit to figure out that `allowEnv` is a field and not an environment variable (since this callout is about environment variables).  How about "Set the `allowEnv` field to..."?  Or maybe move it (and the default behavior blurb) entirely elsewhere?